.rubocop_todo.yml

```diff
@@ -1,28 +1,28 @@
 # This configuration was generated by
 # `rubocop --auto-gen-config`
-# on 2020-05-04 16:48:19 -0700 using RuboCop version 0.73.0.
+# on 2020-07-30 14:01:13 -0500 using RuboCop version 0.73.0.
 # The point is for the user to remove these configuration records
 # one by one as the offenses are removed from the code base.
 # Note that changes in the inspected code, or installation of new
 # versions of RuboCop, may require this file to be generated again.
 
-# Offense count: 3
+# Offense count: 4
 Metrics/AbcSize:
-  Max: 20
+  Max: 21
 
 # Offense count: 2
 # Configuration parameters: CountComments, ExcludedMethods.
 Metrics/MethodLength:
   Max: 15
 
-# Offense count: 3
+# Offense count: 1
 Style/ClassVars:
   Exclude:
     - 'spec/support/authentication_helpers.rb'
 
-# Offense count: 91
+# Offense count: 101
 # Cop supports --auto-correct.
 # Configuration parameters: AutoCorrect, AllowHeredoc, AllowURI, URISchemes, IgnoreCopDirectives, IgnoredPatterns.
 # URISchemes: http, https
 Metrics/LineLength:
-  Max: 122
+  Max: 120
```

Comment:
I think 120 can be in `.rubocop.yml` but not worth holding up this PR.

Reply:
I agree. This was just an autogenerate change though.